docs/source/technical_tutorials/searching.rst

```diff
@@ -2,147 +2,15 @@
-  items = collection.get_items()
-
-
-
-MAAP STAC
-=======================================
-
-MAAP STAC (https://stac.maap-project.org) is dedicated to datasets not accessible via NASA's CMR, such as GEDI Cal/Val datasets, ESA datasets, and user-shared data products.
-
-STAC discovery
----------------------------------------
-
-Users can discover data in MAAP STAC using pystac-client or https://stac-browser.maap-project.org.
-
-API documentation is available here: https://stac.maap-project.org/docs (will return MAAP STAC results).
-
-The general STAC API spec is here: https://api.stacspec.org/v1.0.0-rc.1/core/.
-
-An example of using pystac-client is included above and in `Searching STAC Documentation <search/searching_the_stac_catalog.ipynb>`_.
-
-Data Access via STAC
----------------------------------------
-
-Data assets (files) published to STAC have not moved from the S3 bucket ``s3://nasa-maap-data-store``. ESA data is accessible via public HTTP access. NASA data in S3 is accessible publicly or via role-based bucket policy access.
-
-Users are encouraged to use common AWS S3 libraries for NASA data access, such as Python's boto3.
-
-Each item should have a "data" asset which includes a URL to the data.
-
-For example, https://stac.maap-project.org/collections/BIOSAR1/items/biosar1_roi_lidar58 includes:
-
-.. code-block:: json
-
-  "assets": {
-    "data": {
-      "href": "https://bmap-catalogue-data.oss.eu-west-0.prod-cloud-ocb.orange-business.com/Campaign_data/biosar1/biosar1_roi_lidar58.shx",
-      "type": "application/octet-stream",
-      "roles": [
-        "data"
-      ]
-    },
-  }
-
-NASA's Operational CMR
-=======================================
-
-CMR Discovery
----------------------------------------
-
-Users can discover data NASA's Operational CMR via its publicly accessible API: https://cmr.earthdata.nasa.gov and user interface: https://search.earthdata.nasa.gov.
-
-CMR Search documentation can be found in `Searching Collections <search/collections.ipynb>`_ and `Searching Granules <search/granules.ipynb>`_ and https://cmr.earthdata.nasa.gov/search/site/docs/search/api.html.
-
-CMR Access
----------------------------------------
-
-For all NASA MAAP users, access to NASA'S Operational data is provided via a federated access token. Anything that is in NASA's Operational CMR should be accessed via maap-py so that the federated access token can be used. Users can also access data from LPDAAC (and possibly other DAACs in the future) without maap-py since the workspace should have access via a role-based bucket policy on the LPDAAC cloud bucket.
-
-Anyone can access data through Earthdata Login as well.
-
-Find more documentation about how to access data in CMR in the `Access <accessing.html>`_ section of this documentation.
+MAAP provides access to data in NASA's Operational CMR (Common Metadata Repository, https://cmr.earthdata.nasa.gov) and a MAAP-managed STAC catalog (https://stac.maap-project.org).
```

Comment:
This should probably still be a list, and maybe a short sentence/description for each, so users know which one to go to?

NASA CMR - for NASA produced and curated data.
MAAP CMR - for data not found in NASA CMR, and data produced by MAAP users.